ntc_templates/templates/alcatel_aos_show_interfaces_status.textfsm

```diff
@@ -0,0 +1,22 @@
+Value PORT ((\d+\/?)+)
+Value ADMIN_STATUS (\S+)
+Value AUTO_NEGOTIATION (\S+)
+Value DETECTED_SPEED (\d+)
+Value DETECTED_DUPLEX (\S+)
+Value DETECT_HYBRID (\S+)
+Value DETECTED_PAUSE (\S+)
+Value DETECTED_FEC (\S+)
+Value CONFIGURED_SPEED (\S+)
+Value CONFIGURED_DUPLEX (\S+)
+Value CONFIGURED_HYBRID (\S+)
+Value CONFIGURED_PAUSE (\S+)
+Value CONFIGURED_FEC (\S+)
+Value LINK_TRAP (\S+)
+Value EEE (\S+)
```

Comment:
Should `EEE` be spelled out?
It makes it more clear at the expense of a long capture group name.
Thoughts?

```
Configuring Energy Efficient Ethernet (802.3az)
Energy Efficient Ethernet (EEE) is a protocol to allow ports to operate in idle or low power mode when
there is no traffic to send
```

reference: page 59 of https://www.al-enterprise.com/-/media/assets/internet/documents/n-to-s/omniswitch-aos-release-810r1-network-configuration-user-guide-rev-a-en.pdf